tools/RAiDER/statsPlot.py

```diff
@@ -624,6 +629,7 @@ def create_DF(self):
 
         # seasonal filter
         if self.seasonalinterval:
+            self.seasonalinterval = self.seasonalinterval.split()
```

Comment:
This is not friendly for pre-2000 GPS data. Not that we will do it, but a user might?

Reply:
The year I chose was arbitrary (other than intentionally not being a leap year) and doesn't have any implications on the code as only the time of the day is passed and used as a filter. I am forced to specify a year too, so I cannot go on with just a month and day.